.circleci/config.yml

```diff
@@ -185,7 +185,7 @@ jobs:
           command: |
             for i in {1..5}; do # retry
               URL=https://master.webknossos.xyz/ \
-              yarn test-screenshot && break || s=$?
+              yarn test-screenshot && s=0 && break || s=$?
```

Comment:
@philippotto Unfortunately it is not easily possible in this case, as `-e` is set: `
   -e  Exit immediately if a simple command exits with a non-zero status, unless
       the command that fails is part of an until or  while loop, part of an
       if statement, part of a && or || list, or if the command's return status
       is being inverted using !.` (https://ss64.com/bash/set.html)

Reply:
Oh ok, but then this would work in theory, right?

```
yarn test-screenshot || s=$
if [ $s -eq 0 ]
then
  break
fi
```

But nevermind, the current way is also fine with me :)

Reply:
Only this would work to make sure that `s` is set, and then I personally prefer the current syntax:
```
yarn test-screenshot && s=$? || s=$?
if [ $s -eq 0 ]; then
  break
fi
```